ray-on-gke/user/jupyterhub/jupyter_config/config.yaml

```diff
@@ -0,0 +1,78 @@
+# This file can update the JupyterHub Helm chart's default configuration values.
+#
+# For reference see the configuration reference and default values, but make
+# sure to refer to the Helm chart version of interest to you!
+#
+# Introduction to YAML:     https://www.youtube.com/watch?v=cdLNKUoMc6c
+# Chart config reference:   https://zero-to-jupyterhub.readthedocs.io/en/stable/resources/reference.html
+# Chart default values:     https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/HEAD/jupyterhub/values.yaml
+# Available chart versions: https://jupyterhub.github.io/helm-chart/
+hub:
+  config:
+    Authenticator:
+      admin_users:
+        - user1
+        - user2
+      allowed_users:
+        - user3
+        - user4
+    # ...
+    DummyAuthenticator:
+      password: password
+    JupyterHub:
+      authenticator_class: dummy
+  networkPolicy:
+    enabled: false
+
+prePuller:
+  hook:
+    enabled: false
+
+proxy:
+  chp:
+    networkPolicy:
+      enabled: false
+  traefik:
+    networkPolicy:
+      enabled: false
+
+scheduling:
+  userScheduler:
+    enabled: false
+
+singleuser:
+  nodeSelector:
+    iam.gke.io/gke-metadata-server-enabled: "true"
+  image:
+    name: jupyter/datascience-notebook
+    tag: python-3.10
+  startTimeout: 700
+  profileList:
+    - display_name: "Prof1"
+      description: "Profile 1"
+      default: true
+    - display_name: "Prof2"
+      description: "Profile 2 with extra gpus"
+      kubespawner_override:
+        image: jupyter/datascience-notebook:python-3.11
```

Comment:
Why is this python 3.11 when the default one is 3.10?

Reply:
Changed to just <image-name>:<tag>